libs/components/list-builder/src/lib/modules/list-filters/list-filter-inline-item.component.ts

```diff
@@ -19,27 +19,27 @@ import { ListItemModel } from '@skyux/list-builder-common';
 })
 export class SkyListFilterInlineItemComponent implements OnInit {
   /**
-   * Specifies the name of the filter.
+   * The name of the filter.
    * @required
    */
   @Input()
   public name: string;
 
   /**
-   * Specifies the current value of the filter.
+   * The current value of the filter.
    */
   @Input()
   public value: any;
 
   /**
-   * Specifies the default value of the filter. If the value of the filter
+   * The default value of the filter. If the value of the filter
```

Comment:
@johnhwhite, do you know if the second sentence is correct? I'm not sure what it means for the users to "set the default value" since a default value would presumably be set by default without any user action.

Reply:
@johnhwhite, do you know if the second sentence is correct? I'm not sure what it means for the users to "set the default value" since a default value would presumably be set by default without any user action.

Reply:
That is how it is implemented — the filter is ignored when `value === defaultValue`.